README.md

```diff
@@ -79,3 +79,8 @@ the HTML and asset files ready to be published.
 ## Code of conduct
 
 Please refer to the `alphagov` [code of conduct](https://github.com/alphagov/code-of-conduct). 
+
+## Licence
+
+Unless stated otherwise, the codebase is released under the [MIT License](https://github.com/bahmady/registers-tech-docs/blob/master/LICENSE). This covers both the codebase and any sample code in the documentation.
```

Comment:
I think the URL to the LICENSE file can be relative to the repo so you avoid hardlinking.